tests/client/amo/components/TestAddonReviewList.js

```diff
@@ -151,53 +168,109 @@ describe('amo/components/AddonReviewList', () => {
         .returns(Promise.resolve(reviews));
 
       return loadAddonReviews({ addonSlug, dispatch })
-        .then((loadedReviews) => {
+        .then(() => {
           const expectedAction = setAddonReviews({
             addonSlug, reviews: [fakeReview],
           });
-          assert.deepEqual(loadedReviews, expectedAction.payload.reviews);
           assert.ok(dispatch.called);
           assert.deepEqual(dispatch.firstCall.args[0], expectedAction);
         });
     });
   });
 
+  describe('mapStateToProps', () => {
+    let store;
+
+    beforeEach(() => {
+      store = createStore();
+    });
+
+    function getMappedProps({
+      addonSlug = fakeAddon.slug, params = { addonSlug },
```

Comment:
Yes, it kind of hurts the brain to read it but beats moving the statement down into the function body and wrapping it in `if (params !== undefined) { ...`